.github/workflows/unit-test.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           kubectl: 1.18.4
           kubeval: v0.16.1
           command: |
-            kubeval --strict -d ${{ env.K8S_MANIFEST_DIR }} --ignore-missing-schemas --ignored-filename-patterns kustomization,kind,patch --skip-kinds ClusterConfig
+            kubeval --strict -d ${{ env.K8S_MANIFEST_DIR }} --ignore-missing-schemas --ignored-filename-patterns kustomization,kind,patch,cloudbeat --skip-kinds ClusterConfig
```

Comment:
we shouldn't exclude cloudbeat files, we need to address the warnings instead

Reply:
The cloudbeat files that reside inside the `deploy` folder are not k8s files (plain yml), therefore I don't think there's a need to validate them.